content/en/security/application_security/threats/security_signals.md

```diff
@@ -28,53 +48,100 @@ To filter the security signals in the [Signals Explorer][2], use the search quer
+3. When you declare a new incident, in the **Declare Incident** settings, configure the incident by specifying details such as the severity level and incident commander.
+   1. Estimate impact. Severity levels go from SEV-1 (critical) to SEV-5 (minor impact). When in doubt, always choose the higher severity.
 4. Click **Declare Incident**.
 
 ## Run a workflow
 
 Use [Workflow Automation][5] to manually trigger a workflow for a security signal. 
 
 1. Make sure the workflow you want to run has a security trigger.
-2. On the [Signals Explorer][2] page, select a security signal.
-3. Scroll down to the **What is Workflow Automation** section.
-4. Click **Run Workflow**.
-5. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
+2. On the [Signals Explorer][2] page, open a security signal.
+3. In the **Respond** section, click **Run Workflow**.
+5. In **Run a workflow**, select the workflow you want to run or click **New Workflow**. 
+   - Depending on the workflow you select, you might be required to enter additional input parameters.
+   - If you selected **New Workflow**, Run a Security Workflow opens. To learn more about workflows, see [Workflow Automation][5]. 
 6. Click **Run**.
 
 ## Review and remediate
 
-1. On the [Signals Explorer][2] page, select a security signal.
-2. On the signal side panel, click each of the tabs, such as **Attack Flow**, **Activity Summary**, and **Rule Details**, to review the information.
-3. Review the **Suggested Next Steps**, and take action:
+1. On the [Signals Explorer][2] page, open a security signal.
+2. In the signal details, view each of the sections, such as **What Happened**, **Activity Summary**, and **Rule Details**.
+3. Review the **Next Steps** and take action:
     -  Click **Block all Attacking IPs** (by specific duration or permanently).
-    -  Click **Automated Attacker Blocking** (based on [detection][10] rules).
+    -  Click **Automated Attacker Blocking** (based on [detection][10] rules). This setting requires the Application Security Management **Protect Write** permission.
     -  Click **[Block with Edge WAF][11]**.
 
+## Bulk actions
+
+When you select one or more signals, you can use **Bulk Actions** to perform the following.
+
+### Set state
+
+Set the signal(s) state to **Open**, **Under Review**, or **Archived**.
+
+### Assign the signal to users
+
+Select **Assign selection** and then select the user(s) to assign to the signal. 
+
+Select **Remove all assignments** to reset the signal assignment to none.
+
+### Case management
+
+Datadog [Case Management][6] offers a centralized place to triage, track, and remediate issues detected by Datadog and third-party integrations.
+
+1. On the [Signals Explorer][2] page, select a security signal.
+2. In **Bulk Actions**, select **Create a case**. 
+3. Select **Create a case** or **Add to an existing case** to add the signal to an existing case. 
```

Comment:
I don't think you need to select **Create a case** twice? I think you can probably combine these two into one.

Reply:
You do.